rust/agents/relayer/src/msg/pending_operation.rs

```diff
@@ -125,6 +134,8 @@ pub enum PendingOperationResult {
     Reprepare,
     /// Do not attempt to run the operation again, forget about it
     Drop,
+    /// todo
+    Confirm,
```

Comment:
just noting that idk if this is the best path, i just did it to unblock things, and we'll need to update the doc above